handlers_test.go

```diff
@@ -34,6 +40,19 @@ func TestValidateFlag(t *testing.T) {
 	t.Log("Unimplemented!")
 }
 
+// testCookieContains verifies a string is found in a list of cookies
+func testCookieContains(t *testing.T, want string, got []string) {
```

Comment:
Does it make sense to name this `testCookieExists` as it does not actually check to see if the cookie contains anything but rather checks that it exists, and add a second function `testCookieContains`, potentially in a future PR, which checks to see if a cookie has an expected value set?